persistence-migration/src/main/resources/migration/changelog-v5.7.0.xml

```diff
@@ -23,4 +23,126 @@
     <changeSet id="v5.7.0-4" author="nscuro">
         <modifyDataType tableName="VULNERABLESOFTWARE" columnName="PURL" newDataType="VARCHAR(1024)"/>
     </changeSet>
+
+    <changeSet id="v5.7.0-5" author="nscuro">
+        <sql>
+            CREATE MATERIALIZED VIEW "PORTFOLIOMETRICS_GLOBAL" AS
```

Comment:
This query is identical to `MetricsDao#getPortfolioMetricsForDays`, except:

* It fetches the retention duration directly from the `CONFIGPROPERTY` table.
* It has no checks whether a given project is accessible.